contracts/ICS26Router.sol

```diff
@@ -54,13 +56,15 @@ contract ICS26Router is
 
     /// @notice Initializes the contract instead of a constructor
     /// @dev Meant to be called only once from the proxy
+    /// @param multisigAdmin The address of the multisig admin for IBCUUPSUpgradeable
     /// @param portCustomizer The address of the port customizer
-    function initialize(address portCustomizer) public initializer {
+    function initialize(address multisigAdmin, address portCustomizer) public initializer {
```

Comment:
Yeah, this seems like leaking some information that is not really a concern to the contract. Maybe upgradeAdmin, securityCouncil or something like that?